pkg/visibility/api/rest/pending_workloads_cq.go

```diff
@@ -77,11 +77,21 @@ func (m *pendingWorkloadsInCqREST) Get(ctx context.Context, name string, opts ru
 		localQueuePositions[queueName]++
 
 		if index >= int(offset) {
+			ownerReferences := make([]metav1.OwnerReference, 0, len(wlInfo.Obj.OwnerReferences))
```

Comment:
I would suggest to have a utility function in the package to covert a workload into a pendingworkload. Maybe `buildPendingWorkload(wl, positionInClusterQueue, positionInLocalQueue)`?